package.json

```diff
@@ -9,11 +9,32 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "test": "yarn test:client && yarn test:server",
     "test:client": "cd client && yarn && yarn test",
-    "test:server": "cd server && yarn && yarn test"
+    "test:server": "cd server && yarn && yarn test",
+    "lint": "eslint --ext js,jsx ./",
+    "lint:fix": "eslint --fix --ext js,jsx ./"
   },
   "author": "Sumit Nihalani",
   "license": "ISC",
   "devDependencies": {
-    "concurrently": "^4.1.1"
+    "concurrently": "^4.1.1",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "husky": "^3.0.3",
+    "lint-staged": "^9.2.1"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "lint-staged"
```

Comment:
Can you explain lint-staged? in pre-push and pre-commit?

Reply:
The code for the "lint-staged" key in package.json should be self-explanatory. It runs eslint on all staged js and jsx files. Read about the [lint-staged](https://github.com/okonet/lint-staged#configuration) package to learn more.